db/migrate/20231214133638_add_unique_index_on_invoice_organization_sequential_id.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddUniqueIndexOnInvoiceOrganizationSequentialId < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    add_index :invoices,
+              "organization_id, organization_sequential_id, date_trunc('month'::text, created_at)",
+              name: 'unique_organization_sequential_id',
+              unique: true
```

Comment:
I think that we should add: `where: 'organization_sequential_id IS NOT NULL'` if we will skip this generation for each invoice